extractor_service/main.py

```
@@ -0,0 +1,104 @@
+        try:
+            entities = langextract.extract(text_path, examples=examples)
+            os.remove(text_path)  # Clean up temp file
+            # Convert AnnotatedDocument or other custom objects to dict for JSON serialization
+            if hasattr(entities, "to_dict"):
+                entities = entities.to_dict()
+            elif hasattr(entities, "dict"):
+                entities = entities.dict()
+            elif hasattr(entities, "__dict__"):
+                entities = entities.__dict__
+            return JSONResponse(content={"entities": entities})
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)